api/scripts/import-certification-cpf-cities.js

```diff
@@ -7,13 +7,19 @@
 'use strict';
 const { parseCsv } = require('./helpers/csvHelpers');
 const { knex } = require('../lib/infrastructure/bookshelf');
-const { uniqBy } = require('lodash');
 
 const CITY_COLUMN_NAME = 'Nom_commune';
 const POSTAL_CODE_COLUMN_NAME = 'Code_postal';
 const INSEE_CODE_COLUMN_NAME = 'Code_commune_INSEE';
 const ALTERNATE_CITY_COLUMN_NAME = 'Ligne_5';
 
+function _removeDuplicates(cities) {
+  return cities.filter(function(city) {
+    const key = `${city.INSEECode}${city.postalCode}${city.name}`;
+    return !this.has(key) && this.add(key);
+  }, new Set());
+}
```

Comment:
Ca semble un peu alambiqué, est-ce que reprendre le uniqBy ne serait pas plus clair à la compréhension>

```suggestion
  return _.uniqBy(cities, (city) => `${city.INSEECode}${city.postalCode}${city.name}`);
}
```

Reply:
J'avoue ce bout de code me retourne le cerveau :sweat_smile:

Reply:
Le unidBy ne fonctionne pas que sur un seul attribut ? Parce qu'il faudrait le faire sur le code Insee, postal et le nom de la ville. Si j'ai fait ce bout de code (qui n'est pas de mon invention évidemment) c'est parce qu’à la base, j'étais partis sur un `_.uniqWith(array, _.isEqual)` mais les perf étaient désastreuses.

Reply:
Modifié par un uniqBy.